base_search_mail_content/views/trgm_index_view.xml

```diff
@@ -9,19 +9,19 @@
             <field name="field_id" position="attributes">
                 <attribute name="domain">[('ttype', 'in', ['char', 'text', 'html'])]</attribute>
                 <attribute name="help">"You can either select a field of type 'text', 'char' or 'html'."</attribute>
-          </field>
-      </field>
+            </field>
+        </field>
     </record>
 
-    <record model="ir.ui.view" id="trgm_index_view_tree">
+    <!--<record model="ir.ui.view" id="trgm_index_view_tree">
```

Comment:
Why do you comment this?

Reply:
If it's not needed, it's better to remove it. Git will keep it if needed.